guess.js

```diff
@@ -1,96 +0,0 @@
-const readline = require('readline');
-const rl = readline.createInterface(process.stdin, process.stdout);
-
-function ask(questionText) {
-    return new Promise((resolve, reject) => {
-        rl.question(questionText, resolve);
-    });
-}
-
-choice()
-async function choice() {
-    let start = await ask('human guess or computer guess? ')
-    if (start === 'human guess') {
-       return startComp()
-    }
-    else if (start === 'computer guess') {
-        return startHuman()
-    }
-}
```

Comment:
putting this within a loop, and creating an `else{..}` block, you can be sure that the user input is the proper response that you want